app/helpers/application_helper.rb

```diff
@@ -45,7 +45,7 @@ def set_search_value
   # create javascript-safe url with parameters
   def javascript_safe_url(url)
     # remove utf8=✓& from url to avoid formatting errors
-    URI.decode(url.gsub(/utf8=%E2%9C%93&/, '')).html_safe
+    CGI.unescape(url.gsub(/utf8=%E2%9C%93&/, '')).html_safe
```

Comment:
Yeah, I found it perplexing that they deprecated using `URI.encode` and `URI.decode` for the analogous `CGI` methods.  There is `URI.encode_www_form_component` that does the same thing, but the method name is very misleading, so I just went for the `CGI` methods.

FWIW `URI.decode/encode` has actually been deprecated since Ruby 2.5, and was completely removed in Ruby 3.0.